api/proto/metrics/metrics.proto

```diff
@@ -213,6 +235,7 @@ message deviceMetric {
   // inactive in the controller. 'dormantTimeInSeconds' is subjected to change based
   // of network connectivity.
   uint64 dormant_time_in_seconds = 22;
+  repeated StorageMetric storage_metrics = 23;
```

Comment:
@zedi-pramodh @petr-zededa Why are we sending disk status as part of device metric message? How frequently will it get changed? IMO, it should be part of only device info message but not of metric message.

Reply:
We want to indicate status of disks inside (STORAGE_STATUS_ONLINE or some problem). We do not expect any changes in frequencies right now.
For my understanding info messages is a reaction for configuration changes: when we add app instance, we expect info messages with type of app info to track all statuses from initial to running. Metrics are periodical information about device itself, regardless of config intent. So I cannot see any problem having disk status inside the metrics.

Reply:
@petr-zededa FWIW we currently do have status in the device info message for instance the network info (with the IP addresses etc for the management ports). That information changes as IP addresses come and go. 
So perhaps we should make the status of the datasets be part of the info message (if not already), and have the read/write etc counters in the metrics.

Reply:
Information about the storage system, and the state of the storage system at the moment, are still slightly different things. The purpose of this message is to signal that the storage system and its disks are good or have a problem, not that we have it or what disks we have added. (there is another message for that).

Reply:
@vk-en Discussed with Rishabh and apparently there are two databases, one for the info data (Postgres)  and the other for metrics (time series)
The device or disk status is only processed from Postgres and hence the status info should be part of info message only. Having that in metrics message is of no use.

Please file a separate PR to fix that.

Reply:
@zedi-pramodh [new PR](https://github.com/lf-edge/eve/pull/2529)